### PR TITLE
chore(contracts/allocs): lock omega allocs to omega genesis

### PR DESCRIPTION
**Pull request by kevinhalliday**
Lock omega allocs to those used at deployment.

issue: none

### CODE REVIEW

**Review by corverroos — APPROVED** [review]

File: contracts/allocs/scripts/genallocs.go [hunk 1]
```diff
@@ -32,12 +32,21 @@ func main() {
 	fmt.Println("genallocs succeeded")
 }
 
+// After a persistent network has been deployed, lock it's allocs.
+var locked = map[netconf.ID]bool{
```
Comment (kc1116) [inline contracts/allocs/scripts/genallocs.go:36]:
```suggestion
// locked map of network configuration IDs for persistent networks.
// Once a persistent network has been generated and deployed, its allocations
// should remain unchanged to ensure consistency and stability across the system.
var locked = map[netconf.ID]bool{
```

Comment (kc1116) [inline contracts/allocs/scripts/genallocs.go:36]: Should we move this map to the `allocs` package so it lives with the actual persistent allocs?

**Review by kc1116 — APPROVED** [review]

File: contracts/allocs/scripts/genallocs.go (hunk 1 above)
Comment (kc1116) [reply contracts/allocs/scripts/genallocs.go:36]: And export a func like this 

`func IsLocked(id netconf.ID) bool `

**Review by sideninja — APPROVED** [review]

File: contracts/allocs/scripts/genallocs.go (hunk 1 above)
Comment (kevinhalliday) [reply contracts/allocs/scripts/genallocs.go:36]: yeah I like that

**Pull request merged**